project/Deps.scala

```diff
@@ -203,6 +204,9 @@ object Deps {
     val scalameter =
       "com.storm-enroute" %% "scalameter" % V.scalameterV % "test" withSources () withJavadoc ()
 
+    val scalaCollectionCompat =
+      "org.scala-lang.modules" %% "scala-collection-compat" % V.scalaCollectionCompatV
```

Comment:
what is this for?

Reply:
This is needed to allow `@nowarn` usage on Scala 2.12.x . The annotation was added in the scala std library in the 2.13.2 release.